common/src/main/java/com/hibiscusmc/hmccosmetics/HMCCosmeticsPlugin.java

```diff
@@ -55,6 +56,7 @@ public final class HMCCosmeticsPlugin extends JavaPlugin {
     private static boolean hasModelEngine = false;
     private static boolean onLatestVersion = true;
     private static String latestVersion = "";
+    private static final Map<String, String> emoteMap = new HashMap<>();
```

Comment:
Could this be moved to its own class? It feels more appropriate for it rather than just in the main class.

Reply:
Will do!